### ISSUE TITLE
Unhandled exception in SLF4J

### ISSUE THREAD

**Issue by maxim1um** [ikvmnet/ikvm-maven#32]
Hi, I'm newer to IKVM. 

I do a simple demo for testing poi-tl working performance in IKVM, but When I ran my demo, I got some exceptions from SLF4J. I already check the output dir, "org.slf4j.dll" is contained. And the pom.xml of poi-tl also contains the slf4j dependency. So I'm confused about the exception now and don't know how to fix these exceptions.

Here is the console output

![image](https://github.com/ikvmnet/ikvm-maven/assets/11847049/9086ae87-0290-4de2-aca6-2354e73aaabf)

And this is the terminal output 

![image](https://github.com/ikvmnet/ikvm-maven/assets/11847049/52dafabb-736b-4441-81fb-f922622e022e)

My test demo is 

[https://github.com/maxim1um/ikvm-test.git](https://github.com/maxim1um/ikvm-test.git)

**Comment by wasabii** [comment]
I'm not sure there is an error here. Am I wrong that this is the expected output of referencing a library that depends on SLF4J without including some logger implementation?

**Comment by maxim1um** [comment]
> I'm not sure there is an error here. Am I wrong that this is the expected output of referencing a library that depends on SLF4J without including some logger implementation?

Yes, you're right. I did a simple maven test with poi-tl only on Java, and the warning still existed. It needs to import some logger packages and the logger implementation.

Anyway, thx
